### PR TITLE
Update italian language

### PR DESCRIPTION
**Pull request by bovirus**
@Klocman 

New italian language revision (removed not necessary new line).

Please update the language.

Please make a new test installer rebuild to check italian installer and new italian language.

Please check issues about translation reported here

https://github.com/Klocman/Bulk-Crap-Uninstaller/issues/375

Thanks.

### CODE REVIEW

**Comment by bovirus** [comment]
if you Inno Setup is version 5.x please update Italian language with this .sil file

[Inno Setup 5.6.1 standard (ANSI) 26.12.2020.zip](https://github.com/Klocman/Bulk-Crap-Uninstaller/files/8757009/Inno.Setup.5.6.1.standard.ANSI.26.12.2020.zip)

before to build new installer.

**Comment by JadranR** [comment]
Hi I send Updated Slovenian translation for BCUninstaller 5.3.
Best regards,
Jadran Rudec

File: source/BulkCrapUninstaller/Controls/Settings/CacheSettings.it.resx [hunk 1]
```diff
@@ -118,17 +118,16 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="label1.Text" xml:space="preserve">
-    <value>Il caching può ridurre notevolmente il tempo di ricerca, ma se una applicazione viene modificata BCU potrebbe non accorgersi del cambiamento (ad es. la grandezza)
-NOTA: Per attivare queste opzioni potrebbe essere necessario riavviare BCU.</value>
+    <value>Il caching può ridurre notevolmente il tempo di ricerca, ma se una applicazione viene modificata BCU potrebbe non accorgersi del cambiamento (ad es. la grandezza). NOTA:  per attivare queste opzioni potrebbe essere necessario riavviare BCU.</value>
```
Comment (Klocman) [inline source/BulkCrapUninstaller/Controls/Settings/CacheSettings.it.resx:121]:
```suggestion
    <value>Il caching può ridurre notevolmente il tempo di ricerca, ma se una applicazione viene modificata BCU potrebbe non accorgersi del cambiamento (ad es. la grandezza).
    NOTA:  per attivare queste opzioni potrebbe essere necessario riavviare BCU.</value>
```
The newline can be left here.

File: source/BulkCrapUninstaller/Controls/AdvancedClipboardCopy.it.resx [hunk 2]
```diff
@@ -124,7 +124,7 @@
     <value>Aiuto</value>
   </data>
   <data name="checkBoxUnescape.Text" xml:space="preserve">
-    <value>Unescape</value>
+    <value>Non uscire</value>
```
Comment (Klocman) [inline source/BulkCrapUninstaller/Controls/AdvancedClipboardCopy.it.resx:127]:
```suggestion
    <value>Unescape</value>
```
It's a programming term, I don't think it should be translted.

**Review by Klocman — APPROVED** [review]
Looks good, thank you! 
I'll make a couple of minor fixes and merge it.

**Pull request merged**